### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by ArnaudBand**
Create a popup windows using JavaScript

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -655,6 +778,73 @@ form > textarea {
...
+  .flexText {
+    flex-direction: column;
+  }
+
+  .titleDisplay {
+    color: #172b4d;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+  }
```
Comment (omar-muhamad) [inline style.css]: - [ ] Please fix the popup card styling in the desktop view by centering the position of the card first, then fix the widths, margins, and padding for all popup card elements, and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

File: style.css [hunk 2]
```diff
@@ -81,6 +81,81 @@ body {
...
+  text-align: center;
+  backdrop-filter: blur(4px);
+  background-color: #e5e5e5e6;
+  border-radius: 16px;
+  margin: 30px;
+}
+
+.newElement {
+  background-color: #fff;
+  width: 0%;
```
Comment (omar-muhamad) [inline style.css]: - [ ] Please fix the popup card styling in the mobile view by centering the position of the card first and give it the correct width value and fix the positioning of the line break above the card buttons, and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 2 above)
Comment (Whoistolu) [reply style.css]: The mobile popup looks good now 👍🏽

**Review by gemmen29 — APPROVED** [review]
Hi @B77748, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**